src/components/tests/loop-form.test.tsx

```diff
@@ -15,59 +14,58 @@ const buildWrapper = (listOfPlaces?: string[]) => {
     console.log('test');
   };
 
-  const wrapper = mount(
-    <LoopForm
-      requestLoop={onClick}
-      fleets={['SuperFleet']}
-      listOfPlaces={!!listOfPlaces ? listOfPlaces : fakePlaces()['SuperFleet']}
-    />,
-  );
+  const wrapper = mount(<LoopForm requestLoop={onClick} fleets={[fleetName]} />);
   return wrapper;
 };
 
 describe('form Validation', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
   test('Initial Values', () => {
-    const wrapper = buildWrapper();
+    const wrapper = buildWrapper('SuperFleet');
+    expect(wrapper.find('input[name="targetFleet"]').props().value != '');
     expect(wrapper.find("input[type='number']").props().value).toEqual('');
-    expect(wrapper.findWhere(x => x.name() === 'input' && x.props().value === 'supplies'))
-      .toBeTruthy;
-    expect(
-      wrapper.findWhere(
-        x => x.name() === 'input' && x.props().value === fakePlaces()['SuperFleet'][0],
-      ),
-    ).toBeTruthy;
-
-    expect(
-      wrapper.findWhere(
-        x => x.name() === 'input' && x.props().value === fakePlaces()['SuperFleet'][1],
-      ),
-    ).toBeTruthy;
+    expect(wrapper.findWhere(x => x.name() === 'input' && x.props().value != '')).toBeTruthy();
+    expect(wrapper.find('input[name="startLocation"]').props().value).not.toEqual(
+      wrapper.find('input[name="finishLocation"]').props().value,
+    );
     wrapper.unmount();
   });
 
+  test('Successful Request', () => {
+    const spy = jest.spyOn(console, 'log');
```

Comment:
This seems like a questionable spy. `console.log` can be called pretty much anywhere.